### ISSUE TITLE
Seem like EfficientNetV2M has some training issues

### ISSUE THREAD

**Issue by macsunmood** [leondgarse/keras_efficientnet_v2#12]
The accuracy of EfficientNetV2M (with any pretrained weights) while it's training is extremely low (other models perform well).

**UPD:** I've just found out that the same issue applies to EfficientNetV2L as well..

**Comment by leondgarse** [comment]
This actually happened to me earlier, using my [Keras_insightface](https://github.com/leondgarse/Keras_insightface) project, that `V2S` performs better that `V2M`. In the first 4 epochs, their training accuracy are like:
```py
"V2S_loss": [17.4384, 14.9046, 20.9333],
"V2S_acc": [0.3089, 0.6853, 0.8034, 0.8804],
"V2M_loss": [17.4384, 14.9046, 20.9333, 17.0123],
"V2M_acc": [0.1652, 0.5790, 0.7463, 0.8571],
```
This is a basic `cifar10` train test result using `V2S` and `V2M` I just finished:
![Selection_403](https://user-images.githubusercontent.com/5744524/132818833-b524ed3f-5c83-4987-9018-8d737f2ed19a.png)

We can see the diffs from `V2M` / `V2L` from `V2S`, that they have additional blocks after the last `strides=2` block. Maybe they are just harder to train, or need harder regularizer? I'm not sure about this.

**Comment by macsunmood** [comment]
This is interesting, I found that the issue occures as sensitively noticeable with very small batch sizes (4, for example).

**Comment by leondgarse** [comment]
Ya, but for small batchs, `BN` layers will not work well, as described in many papers like [Arxiv: Group Normalization](https://arxiv.org/pdf/1803.08494.pdf). I may run more experimentss on `V2M` next week.

**Comment by leondgarse** [comment]
Just got time for testing `V2M` model recently... I ran some training using my [Keras_insightface](https://github.com/leondgarse/Keras_insightface) project, and the result comparing `V2S` is not bad for me:
![Selection_369](https://user-images.githubusercontent.com/5744524/135441802-7d4125f7-a1b4-415e-b8e8-23861bd600a5.png)
The green and orange line are `V2M` results. Orange line using `drop_connect_rate=0.2` and green line using `drop_connect_rate=0.3`.
